frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -104,6 +107,28 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
     }
   };
 
+  const submit = (event: React.FormEvent<EventTarget>) => {
+    event.preventDefault();
+    setProgress(true);
+
+    if (isNoProvionerSC) {
+      const [pvsData, pvsLoaded, pvsLoadError] = pvsList;
+      if ((pvsLoadError || pvsData.length === 0 || !storageClass) && pvsLoaded) {
```

Comment:
when the storage class is not set why do we want to make a request?

Reply:
If it's loaded & we know the pvs are present we should get getMinSizePVCapacity otherwise we set default size. This might reduce the complexity of statement.

Reply:
That is what I am doing. Refer https://github.com/openshift/console/pull/6642/files#diff-ecff7031320d48d9e801ba99a2fb7188R120. 
Request will not go, as there is check for storage class.

Reply:
We are checking storage class over here itself. Why need to check again?

https://github.com/openshift/console/pull/6642/files#diff-ecff7031320d48d9e801ba99a2fb7188R114

Reply:
ALso do we need to update the value while doing add capacity & replace the existing values?